app/src/main/java/com/getsensibill/spendmanager/demo/subclass/SubclassKotlinActivity.kt

```diff
@@ -0,0 +1,119 @@
+package com.getsensibill.spendmanager.demo.subclass
+
+import android.os.Bundle
+import com.getsensibill.spendmanager.demo.databinding.ActivitySubclassKotlinBinding
+import com.getsensibill.web.data.UiFinishReason
+import com.getsensibill.web.data.configuration.NavigationIntent
+import com.getsensibill.web.data.configuration.ProgrammaticTheme
+import com.getsensibill.web.data.models.Brand
+import com.getsensibill.web.ui.WebUiActivity
+import com.getsensibill.web.ui.WebUiFragment
+import com.getsensibill.web.ui.WebUiNetworkErrorFragment
+import timber.log.Timber
+
+/**
+ * Subclassing the WebUiActivity
+ * <p>
+ * When you subclass the WebUiActivity, you do not need to override anything for it to work.
+ * It will use all the defaults provided by the [WebUiActivity] and the [WebUiFragment].
+ *
+ * If you are using your own view, then its important to override the [callSetContentView] and
```

Comment:
`view` - do you think we should say "layout" instead?

Reply:
updated